content/webapp/contexts/ConceptPageContext/concept.config.tsx

```diff
@@ -19,17 +19,60 @@ type ConceptConfig = {
   worksBy: ConceptSection;
   worksAbout: ConceptSection;
   worksIn: ConceptSection;
-  contributors: ConceptSection & {
+  collaborators: ConceptSection & {
     maxCount?: number;
   };
   relatedTopics: ConceptSection & { excludedTopics?: ConceptType[] };
 };
 
+export const defaultConceptConfig: ConceptConfig = {
+  displayName: {
+    display: false,
+  },
+  partOf: {
+    display: false,
+  },
+  sourcedDescription: {
+    display: false,
+  },
+  fieldOrArea: {
+    display: false,
+  },
+  imagesBy: {
+    display: true,
+    label: 'Images by this concept',
+  },
+  imagesAbout: {
+    display: true,
+    label: 'Images about this concept',
+  },
+  imagesIn: {
+    display: false,
+  },
+  worksBy: {
+    display: true,
+    label: 'Works by this concept',
+  },
+  worksAbout: {
+    display: true,
+    label: 'Works about this concept',
+  },
+  worksIn: {
+    display: false,
+  },
+  collaborators: {
+    display: false,
+  },
+  relatedTopics: {
+    display: false,
+  },
+};
+
 // The API response hasn't been curated for the front-end in order to make as much content available to
 // the wider public as possible. Here we define a config for each of the individual displayed concept types
 // to simplify the front end templates (fewer hard-to-read conditionals). This will hopefully also help
 // with explaining to non-devs how and why different concepts are expected to appear
-export function makeConceptConfig(concept: Concept): ConceptConfig | undefined {
+export function makeConceptConfig(concept: Concept): ConceptConfig {
```

Comment:
np, we could just pass the type here, no need for the whole object

Reply:
we need parts of it for the labels – would it be better to just pull out the bits we need?